docs/website/docs/community/index.md

```diff
@@ -1,17 +1,34 @@
 # Community projects
 
+Projects built by community members:
+
+*   The [SHARK](https://github.com/nod-ai/SHARK) project from
+    [nod.ai](https://nod.ai/) uses a forked version of IREE
+    ([SHARK-Runtime](https://github.com/nod-ai/SHARK-Runtime)), offering
+    highly tuned performance on a large corpus of machine learning programs.
```

Comment:
@powderluv Is this okay to include here (https://iree-org.github.io/iree/community/)? Want this left off, or the wording changed?